docs/blog-2.md

```diff
@@ -0,0 +1,209 @@
+
+# `git machete` strikes again! Traverse the git (rebase) jungle even faster with v2.0+
+
+
+## Intro
+
+Good news - git machete 2.0 has been released!
+
+For those of you not familiar with this nifty tool, I recommend having a quick look at the [previous part of this series (link)](https://virtuslab.com/blog/make-way-git-rebase-jungle-git-machete)
+or at least scrolling through the screenshots in that blog post to get a TL;DR kind of understanding of what git machete actually does.
+
+In general, git machete is a tool that helps manage multiple branches within a git repository and thus promotes the creation of smaller pull requests that are (usually) easier to review.
+Just recall the last time you had to review a giant like this (diff stats shown of course only for a few first files):
+
+![Too many files in a single PR...](files-changed.png)
+
+You can get the latest git machete release directly from [the git machete repo (github.com/VirtusLab/git-machete)](https://github.com/VirtusLab/git-machete):
+
+```
+$ git clone https://github.com/VirtusLab/git-machete.git
+$ cd git-machete
+$ sudo make install
+```
+
+`make install` copies the `git-machete` Python 2.7 executable to `/usr/local/bin` and sets up the corresponding Bash completion script in `/etc/bash_completion.d`.
+
+Some of the latest improvements (especially automatic discovery of branch dependency tree) have been suggested in the [Reddit discussion (link)](https://redd.it/8625a6) for the previous part of the series -
+many thanks for the feedback.
+Also, special thanks to the GitHub user [sprudent](https://github.com/sprudent), who raised the issue of git machete crashing when run from a git submodule.
+Other recent tweaks to git machete were introduced simply to make day-to-day use of the tool even more convenient.
+
+## Demo repository setup
+
+As in the first part of the series, there is a dedicated script that sets up a demo repository with a couple of branches -
+you can download it [directly from GitHub (link)](https://raw.githubusercontent.com/PawelLipski/git-machete-blog-2/master/sandbox-setup-2.sh).
```

Comment:
Pls remove dependency on `PawelLipski/git-machete-blog*` repos, instead refer to the script directly from this repository